yospace/src/integration/YospaceAdHandler.ts

```diff
@@ -144,6 +144,13 @@ export class YospaceAdHandler {
             this.advertStartListener = undefined;
         }
 
+        if (this.currentAdBreak === undefined) {
+            const missingAdBreak = advert.broker.getAdBreakForAdvert(advert);
```

Comment:
Nit-pick: is `advert.broker` always the same as `this.yospaceManager.sessionManager`?

Reply:
I'm not 100% sure of that, since I only found this `broker` by debugging and did not find any additional documentation for it. It did seem to match consistently throughout the testing.